docs/content/what-is-a-list-filter.md

```diff
@@ -25,7 +25,7 @@ WorkflowType = "main.YourWorkflowDefinition" and ExecutionStatus != "Running" an
 
 [More example List Filters](#example-list-filters)
 
-A List Filter contains Search Attribute keys, Search Attribute values, and Operators.
+A List Filter contains Search Attribute names, Search Attribute values, and Operators.
```

Comment:
Why change this to "names"?

A Search Attribute is a key, value pair right?
I am OK with "key name" as long as it is consistent.

See comment below about use of "key" or "key name" instead of "field".